rllib/evaluation/postprocessing.py

```diff
@@ -57,13 +57,13 @@ def compute_advantages(rollout: SampleBatch,
         rollout[Postprocessing.ADVANTAGES] = discount(delta_t, gamma * lambda_)
         rollout[Postprocessing.VALUE_TARGETS] = (
             rollout[Postprocessing.ADVANTAGES] +
-            rollout[SampleBatch.VF_PREDS]).copy().astype(np.float32)
+            rollout[SampleBatch.VF_PREDS]).astype(np.float32)
```

Comment:
Hmm can you look into why the .copy() was added? I'm a little concerned it was for working around some bug.

Reply:
Has always been there since 0.6.x
These copies really don't make sense here and we should remove them:
```
print(timeit.timeit("d = (a + b).copy()", number=100000, setup="import numpy as np; a = np.zeros(shape=(800,), dtype=np.float32); b = np.ones(shape=(800,), dtype=np.float32)"))
print(timeit.timeit("c = (a + b)", number=100000, setup="import numpy as np; a = np.zeros(shape=(800,), dtype=np.float32); b = np.ones(shape=(800,), dtype=np.float32)"))
```

Reply:
~ twice as fast w/o:
```
0.1241755
0.0717351
```